db/migrations/2019_11_19_000000_add_api_privilege.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Engelsystem\Migrations;
+
+use Engelsystem\Database\Migration\Migration;
+
+class AddApiPrivilege extends Migration
+{
+    /**
+     * Run the migration
+     */
+    public function up()
+    {
+        $connection = $this->schema->getConnection();
+        // Add api permissions
+        $connection->table('Privileges')->insert(
```

Comment:
I Would format it as
```php
$this->schema->getConnection()
->table('Privileges')
->[...]
```
but thats only my personal preference

Reply:
I don't. But you could collapse it in this case with just a single insert.